app/controllers/Application.scala

```diff
@@ -87,11 +105,11 @@ class Application @Inject()(val akkaSystem: ActorSystem, ws: WSClient, videoServ
     }
   }
 
-  def videoStrip(w: Option[Int], h: Option[Int], callback: Option[String], rotate: Option[Int], aspectRatio: Option[Double]) = Action.async(BodyParsers.parse.temporaryFile) { request =>
+  def videoStrip(w: Option[Int], h: Option[Int], callback: Option[String], rotate: Option[Int], aspectRatio: Option[Double]) = Action.async(parse.temporaryFile) { request =>
     val width = w.getOrElse(320)
     val height = h.getOrElse(180)
 
-    implicit val videoProcessingExecutionContext: ExecutionContext = akkaSystem.dispatchers.lookup("video-processing-context")
+    val videoProcessingExecutionContext: ExecutionContext = akkaSystem.dispatchers.lookup("video-processing-context")
```

Comment:
removed implicit classifier from custom execution contexts as:
- they were causing ambiguous implicit resolution errors due to
`import scala.concurrent.ExecutionContext.Implicits.global`
- they are hard referenced at the call sites